src/engine/source/kvdb2/include/kvdb2/kvdbSpace.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef _KVDBSPACE_H
```

Comment:
I think it should be `_KVDB_SPACE_H`

Reply:
Will be fixed in later documentation step.